### DISCUSSION TITLE
Rslogix 500 Pro

### DISCUSSION THREAD

**Discussion by evandro665** [community/community#102063] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Boa tarde Galera, estou precisando de uma ajudar para retirar senha do acesso Rslogix 500 Pro.  
![Rslogix500](https://github.com/community/community/assets/67564697/1a3f82e2-1c21-4694-8c3b-fe158a8713eb)

**Comment by queenofcorgis** [comment]
Hi @evandro665 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Rslogix 500 Pro. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by evandro665** [reply]
Good afternoon, thank you very much for your attention
